03week/towersOfHanoi.js

```diff
@@ -19,24 +19,104 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+function reset() {
+  stacks.a = [4, 3, 2, 1];
+  stacks.b = [];
+  stacks.c = [];
+}
+
+function movePiece(source, dest) {
   // Your code here
+  //Check for the legal move
+  //move the last piece of source stack to destination stacks
+  //check for the win
+  if (isLegal(source, dest)) {
+    const piece = stacks[source].pop();
+    stacks[dest].push(piece);
```

Comment:
this function should only be moving the piece, not checking for legality or checking for a win